blog/_posts/2015-12-2-includeos-is-now-open-source.md

```diff
@@ -0,0 +1,38 @@
+
+
+## C++ is a new language
+If you haven't looked at C++ after 2011, you should look now -  "It feels like a new language" (Bjarne Stroustrup). While you can always argue that adding new feature won't necessarily make the language any better, it most definitely makes it more powerful. Here are a couple of really nice new language features we're using in IncludeOS:
+
+1. Real Lambdas. Not function pointers, not just call operators, but lambdas just like you know and love from functional languages.
+2. C++11 implementation of delegates (i.e. pointers to a member function in a class instance), which are exactly as fast as function pointers. We're using these everywhere. Reading [the implementation](https://github.com/hioa-cs/IncludeOS/blob/master/api/utility/delegate.hpp) is a great way to have your brain explode, but they work like magic.
+4. User defined literals, and std::literals for units. You can set timers by passing in `100ms`, `10s` or `50min` as parameters to the same function. 
+3. A lot of wisdom, packed into concise [guidelines from the Jedi Council](https://github.com/isocpp/CppCoreGuidelines).
+
+Obviously, with C++ you can mess up the memory of your own program if you want to, but compared to C you have so many more tools for writing encapsulated and type-safe code. If you haven't looked at C++ since 2011, you should really look now. It "feels like new language" (Bjarne Stroustrup).
+
+## Thanks for the warm welcome!
+Opening up a repository after over a year, to much expectation from people you look up to, is pretty scary. There are so many things I'd like to improve, fix and add, but at some point you just have to let go and let it rip. The hope was that opening up would help the process - and indeed I got a pull request fixing all my typos the very next day. The inclusive response from [The Jedi Council](https://isocpp.org/), and the immediate welcome on twitter from the [Mirage](https://mirage.io/) guys have made us feel like family, more than competition, which is what we'd like to be. We hope to be able to contribute something back to the Unikernel effort as a whole.
```

Comment:
Yay for open source!  Same comment as previously re: Jedi Council.

minor typos:
- twitter -> Twitter (you can link to your tweet here. It's: https://twitter.com/AlfredBratterud/status/671809686820143104 ... in fact, I'd actually *embed* the tweet in the post)
- Mirage -> MirageOS